rhel/repositoryscanner.go

```diff
@@ -318,16 +322,31 @@ func extractBuildNVR(ctx context.Context, dockerfilePath string, buf *bytes.Buff
 	}
 	n, ok := ls[comp]
 	if !ok {
-		return "", "", fmt.Errorf("dockerfile missing expected label %q", comp)
+		return "", "", missingLabel(comp)
 	}
 	a, ok := ls[arch]
 	if !ok {
-		return "", "", fmt.Errorf("dockerfile missing expected label %q", arch)
+		return "", "", missingLabel(arch)
 	}
 	v, r := parseVersionRelease(filepath.Base(dockerfilePath))
 	return fmt.Sprintf("%s-%s-%s", n, v, r), a, nil
 }
 
+var errBadDockerfile = errors.New("bad dockerfile")
+
+type missingLabel string
```

Comment:
As this fulfills the `Error` interface, should it be prefixed `err` for easy recognition?

Reply:
As it's an unexported type, I don't think it matter enough to have you hit the button again.